src/main.ts

```diff
@@ -114,6 +118,11 @@ export default class LLMPlugin extends Plugin {
 		await this.checkForAPIKeyBasedModel();
 		this.registerRibbonIcons();
 		this.registerCommands();
+		this.messageStore = new MessageStore();
+		this.settings.currentIndex = -1;
+		this.messageStore.setMessages([]);
+		console.log(this.messageStore.getMessages());
```

Comment:
The console log will get us in trouble in the review process (although we will release the previous version & this change set will come out with the first release after we get approved).

Reply:
Ah, yea I can delete this. Was left over from debugging